### PR TITLE
Cmake docs and other

### PR DESCRIPTION
**Pull request by SebastianBoe**

### CODE REVIEW

File: .shippable.yml [hunk 1]
```diff
@@ -4,10 +4,10 @@ compiler: gcc
 
 env:
     global:
-        - SDK=0.9.1
+        - SDK=0.9.2
         - SANITYCHECK_OPTIONS=" --inline-logs -R"
         - SANITYCHECK_OPTIONS_RETRY="${SANITYCHECK_OPTIONS} --only-failed --outdir=out-2nd-pass"
-        - ZEPHYR_SDK_INSTALL_DIR=/opt/sdk/zephyr-sdk-0.9.1
+        - ZEPHYR_SDK_INSTALL_DIR=/opt/sdk/zephyr-sdk-0.9.2-rc4
```
Comment (nashif) [inline .shippable.yml]: still not ready, will fail on many architectures due to some issues in Qemy, rc5 will come soon.

Comment (SebastianBoe) [reply .shippable.yml]: Thanks for the heads up, I'll split up the patchset to get this merged then.

File: doc/introduction/introducing_zephyr.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ resource-constrained systems: from simple embedded environmental sensors and LED
 wearables to sophisticated smart watches and IoT wireless gateways.
 
 The Zephyr kernel supports multiple architectures, including ARM Cortex-M, Intel
-x86, ARC, NIOS II, Tensilica Xtensa and RISC V. The full list of supported
+x86, ARC, NIOS II, Tensilica Xtensa and RISC-V. The full list of supported
```
Comment (dbkinder) [inline doc/introduction/introducing_zephyr.rst]: add the comma before and:

     Tensilica Xtensa, and RISC-V

File: doc/getting_started/getting_started.rst [hunk 3]
```diff
@@ -70,7 +70,7 @@ To build an example application follow these steps:
```
Comment (dbkinder) [inline doc/getting_started/getting_started.rst]: while you're in there, how about amending the previous line to read (for clarity):

     #. Navigate to the main project directory (where you cloned the Zephyr source from GitHub):

File: doc/introduction/introducing_zephyr.rst (hunk 2 above)
Comment (SebastianBoe) [reply doc/introduction/introducing_zephyr.rst]: Done.

File: doc/getting_started/getting_started.rst (hunk 3 above)
Comment (SebastianBoe) [reply doc/getting_started/getting_started.rst]: Done.

File: doc/getting_started/getting_started.rst [hunk 4]
```diff
@@ -79,35 +79,49 @@ To build an example application follow these steps:
...
       $ source zephyr-env.sh
 
-#. Build the :ref:`hello_world` example project, enter:
+#. Generate project files for the :ref:`hello_world` example, enter:
 
    .. code-block:: console
 
       $ cd $ZEPHYR_BASE/samples/hello_world
-      $ make
+      $ mkdir build
```
Comment (dbkinder) [inline doc/getting_started/getting_started.rst]: Oh dear, so this means we'll need to find every commandline example where have "make xxx" and change it to mkdir/cd/cmake/cd/ninja throughout the documentation, right?

File: doc/getting_started/getting_started.rst [hunk 5]
```diff
@@ -79,35 +79,49 @@ To build an example application follow these steps:
...
+
+   $ cd $ZEPHYR_BASE/samples/hello_world/build
+   $ ninja
 
-The above invocation of make will build the :ref:`hello_world` sample application
-using the default settings defined in the application's Makefile. You can
-build for a different board by defining the variable BOARD with one of the
-supported boards, for example:
+You can build for a different board by defining the variable BOARD
+with one of the supported boards, for example:
```
Comment (dbkinder) [inline doc/getting_started/getting_started.rst]: Is there place/command we can give folks to provide a list of the supported board names (such as arduino_101) in this  new cmake world?  (Today I can do a "make help" to see board names).

File: doc/getting_started/getting_started.rst (hunk 4 above)
Comment (SebastianBoe) [reply doc/getting_started/getting_started.rst]: Yes, or the slightly shorter

cmake -H. -Bbld && cmake --build bld

File: doc/getting_started/installation_linux.rst [hunk 6]
```diff
@@ -52,23 +52,32 @@ Install the required packages in a Ubuntu host system with:
 
 .. code-block:: console
 
-   $ sudo apt-get install git make gcc g++ ncurses-dev gperf \
-	 doxygen dfu-util device-tree-compiler python3-ply python3-pip
+   $ sudo apt-get install --no-install-recommends git ninja-build gperf \
+	 doxygen dfu-util device-tree-compiler \
+     python3-ply python3-pip python3-setuptools xz-utils file
```
Comment (dbkinder) [inline doc/getting_started/installation_linux.rst]: would look getter with these two lines aligned...

**Review by dbkinder — CHANGES_REQUESTED** [review]
one more tweak and otherwise doc looks ok

File: doc/getting_started/installation_linux.rst (hunk 6 above)
Comment (SebastianBoe) [reply doc/getting_started/installation_linux.rst]: Done

**Comment by SebastianBoe** [comment]
@nashif , @dbkinder : All review comments have been addressed, this is ready for approval.

**Review by nashif — APPROVED** [review]

**Pull request merged**